api/src/test/java/org/openmrs/util/OpenmrsUtilTest.java

```diff
@@ -63,6 +72,19 @@
 	        OpenmrsConstants.GLOBAL_PROPERTY_DEFAULT_PATIENT_IDENTIFIER_VALIDATOR,
 	        OpenmrsConstants.LUHN_IDENTIFIER_VALIDATOR);
 	
+	private ConceptService cs;
+	private final String separator = FileSystems.getDefault().getSeparator();
+	
+	@Before
+	public void setup() {
+		cs=Context.getConceptService();
+	}
+	private File createFileAtRelativePath(String relativePath) {
```

Comment:
What do you mean by `createFileAtRelativePath` ? Doesn't `createFileInUserDirectory` sound better?